packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -301,7 +305,7 @@ contract OptimisticGovernor is Module, Lockable {
     }
 
     function _sync() internal {
-        skinnyOptimisticOracle = _getOptimisticOracle();
+        optimisticOracle = _getOptimisticOracle();
         finalFee = _getStore().computeFinalFee(address(collateral)).rawValue;
```

Comment:
Is finalFee used anywhere? Can we drop it?

Reply:
Looks like we can drop it now.